app/views.py

```diff
@@ -64,6 +62,15 @@ def settings():
     image_file = url_for('static', filename='resources/images/profile_pics/' + current_user.image_file)
     return render_template('settings.html', title=' | Settings', image_file=image_file, form=form)
 
+@app.route('/settings/knewbieID', methods=['GET', 'POST'])
```

Comment:
Should only allow GET request since there is no need to POST to this link.
```suggestion
@app.route('/settings/knewbieID')
```